sdk/lib/opentelemetry/sdk/configurator.rb

```diff
@@ -27,7 +27,7 @@ def initialize
         @text_injectors = nil
         @span_processors = []
         @use_mode = USE_MODE_UNSPECIFIED
-        @tracer_provider = Trace::TracerProvider.new
+        @resource = OpenTelemetry::SDK::Resources::Resource.create
```

Comment:
We should be able to default this to the Telemetry SDK Resource. We're required to have one by the spec and this would be a good place to initialize it. I talk about it more in a comment further down.

Reply:
Changes regarding this comment here https://github.com/open-telemetry/opentelemetry-ruby/pull/263/commits/3cbcb13493fc7522dadb94baa7a0bf4806f929ac

Not sure if this follows the spec correctly as I wasn't able to find the specific parts around this requirement.